src/pages/SwapV3/index.tsx

```diff
@@ -0,0 +1,495 @@
+                      <Trans>Limit</Trans>
+                    </Text>
+                  </Tab>
+                </TabWrapper>
+              </TabContainer>
+
+              <SwapFormActions>
+                <Tutorial
+                  type={TutorialType.SWAP}
+                  customIcon={
+                    <StyledActionButtonSwapForm>
+                      <TutorialIcon />
+                    </StyledActionButtonSwapForm>
+                  }
+                />
+                {chainId !== ChainId.ETHW && (
+                  <MobileTokenInfo
+                    currencies={isSwapPage ? currencies : currenciesLimit}
+                    onClick={() => onToggleActionTab(TAB.INFO)}
+                  />
+                )}
+                <ShareButtonWithModal
+                  title={t`Share this with your friends!`}
+                  url={shareUrl}
+                  onShared={() => {
+                    mixpanelHandler(MIXPANEL_TYPE.TOKEN_SWAP_LINK_SHARED)
+                  }}
+                />
+                <StyledActionButtonSwapForm
+                  active={activeTab === TAB.SETTINGS}
+                  onClick={() => onToggleActionTab(TAB.SETTINGS)}
+                  aria-label="Swap Settings"
+                >
+                  <MouseoverTooltip
+                    text={!isExpertMode ? <Trans>Settings</Trans> : <Trans>Advanced mode is on!</Trans>}
+                    placement="top"
+                    width="fit-content"
+                  >
+                    <span id={TutorialIds.BUTTON_SETTING_SWAP_FORM}>
+                      <TransactionSettingsIcon fill={isExpertMode ? theme.warning : theme.subText} />
+                    </span>
+                  </MouseoverTooltip>
+                </StyledActionButtonSwapForm>
+              </SwapFormActions>
+            </RowBetween>
+
+            <RowBetween>
+              <Text fontSize={12} color={theme.subText}>
+                {isLimitPage ? (
+                  <Trans>Buy or sell any token at a specific price</Trans>
+                ) : (
+                  <Trans>Buy or sell any token instantly at the best price</Trans>
+                )}
+              </Text>
+            </RowBetween>
+
+            {chainId !== ChainId.ETHW && !isSolana && (
+              <RowBetween>
+                <PairSuggestion
+                  ref={refSuggestPair}
+                  onSelectSuggestedPair={onSelectSuggestedPair}
+                  setShowModalImportToken={setShowingPairSuggestionImport}
+                />
+              </RowBetween>
+            )}
+
+            <AppBodyWrapped data-highlight={shouldHighlightSwapBox} id={TutorialIds.SWAP_FORM}>
+              {activeTab === TAB.SWAP && (
+                <PopulatedSwapForm
+                  routeSummary={routeSummary}
+                  setRouteSummary={setRouteSummary}
+                  goToSettingsView={() => setActiveTab(TAB.SETTINGS)}
+                />
+              )}
```

Comment:
This will make swap form reset when click tab. You can use display none to hide it